rest/data/repotracker.go

```diff
@@ -31,6 +34,16 @@ func (c *RepoTrackerConnector) TriggerRepotracker(q amboy.Queue, msgID string, e
 	if len(branch) == 0 {
 		return nil
 	}
+	adminSettings, err := evergreen.GetConfig()
+	if err != nil {
+		return errors.Wrap(err, "error retrieving admin settings")
+	}
+	if adminSettings.ServiceFlags.RepotrackerDisabled {
+		grip.InfoWhen(sometimes.Percent(evergreen.DegradedLoggingPercent), message.Fields{
+			"message": "github push events triggering repotracker is disabled",
```

Comment:
Change message to "repotracker is disabled". Change the error just below this too.
Add another field: "source": "github hooks"